doc/create_doc_windows.ps1

```diff
@@ -0,0 +1,29 @@
+docker pull ghcr.io/ansys/pydpf-composites:latest
+docker pull ghcr.io/ansys/mapdl:latest
+
+if ($Env:ANSYSLMD_LICENSE_FILE -ne $null)
+{
+    ">> ANSYSLMD_LICENSE_FILE=" + $Env:ANSYSLMD_LICENSE_FILE
+}
+else
+{
+    ">> ANSYSLMD_LICENSE_FILE is not set. Use it if the license server is not otherwise defined."
```

Comment:
I think the `ANSYSLMD_LICENSE_FILE` is _required_ (there's no way to specify it otherwise), since that's how it's passed into the PyMAPDL + PyDPF Composites containers.